drop-storage/src/types.rs

```diff
@@ -3,6 +3,66 @@ use serde::Serialize;
 type TransferId = uuid::Uuid;
 type FileId = String;
 
+#[derive(Debug, Serialize)]
+#[serde(tag = "state")]
+pub enum PathStateEventData {
```

Comment:
This should be split into two types, one for incoming and the other for outgoing IMHO

Reply:
done :+1: